src/graphql/mutations/addUserToTeamMutation.ts

```diff
@@ -0,0 +1,45 @@
+import { gql } from '@apollo/client';
+
+export const ADD_USER_TO_TEAM_MUTATION = gql`
+  mutation addUserToTeam($data: AddUserToTeamInput!) {
+    addUserToTeam(data: $data) {
+      id
+      firstName
+      lastName
+      github
+      telegram
+      discord
+      score
+      country
+      city
+      avatar
+      isAdmin
+      courses {
+        id
+        name
+      }
+      email
+      courseIds
+      teamIds
+      teams {
+        id
+        password
+        number
+        courseId
+        socialLink
+        memberIds
+        members {
```

Comment:
memberIds, members - are those two fields necessary at the same time?
Also, I'd ask you not to use nested entities if it's not really needed to decrease amount of database calls.

Reply:
I'll take this into account